src/types.jl

```diff
@@ -0,0 +1,154 @@
+abstract AbstractODE
+
+
+type ExplicitODE <: AbstractODE
```

Comment:
Needs doc, in particular as there are implicit/explicit solvers and implicitly/explicitly defined ODEs.